### ISSUE TITLE
Remove timestamps from storage and event logs

### ISSUE THREAD

**Issue by mds1** [dcgtc/dgrants#503]
This was implemented in #497, but it turns out we don't actually need these helpers because we can get the timestamps from the subgraph + event listeners as needed cc @gdixon @thelostone-mc

**Comment by mk1r** [comment]
Are we effectively reverting this PR? or is there stuff we need to keep, most of it seems related to timestamps

**Comment by mds1** [comment]
Basically reverting it, but we aren't doing this until after GR12 when we flesh out protocol specs/contracts a bit more, because it's already done and the added gas costs on Polygon for GR12 are negligible

**Comment by mk1r** [comment]
@mds1 ah I see, cool

**Comment by mds1** [comment]
@mk1r noticed you were assigned to this, but leaving this unassigned for now since there's currently no plans to work on this

**Comment by gdixon** [comment]
I think we need to keep these changes in place because although in most cases we can use the `timestamps` provided by the `subgraph` or the `event`, when we initially filter the RPC for content between two given blocks (when we don't have a subgraph url), then there is no clear-cut way of retrieving the `block`/`timestamps`. Using `tx.args.time` makes this a lot easier 🙌 

@mds1 - how do you feel about closing this out?

**Comment by mds1** [comment]
>  then there is no clear-cut way of retrieving the `block`/`timestamps`

This is the main thing I'd want to verify before closing this out. When using an event listener with ethers, it's still listening for each block under the hood. (And possibly something for query filters where it already retrieved block info). Meaning that I suspect we might have already queried for block metadata (i.e. `timestamp`) and it's just discarded/not exposed cleanly in our current approach, but I'm not certain of this

In other words, before closing this out we should verify that there _isn't_ a way to rewrite our event queries to get timestamp for free without an extra RPC call
